config/log.syslog.ini

```diff
@@ -1,9 +0,0 @@
-[general]
```

Comment:
Removing this file seems incorrect.

Reply:
It's an anacronism, the new syslog plugin uses syslog.ini.

Reply:
Gotcha